pubspec.yaml

```diff
@@ -11,6 +11,7 @@ dependencies:
   collection: ^1.15.0
   path: ^1.8.0
   term_glyph: ^1.2.0
+  tuple: ^2.0.0
```

Comment:
Do we need this dependency? Could we just copy-paste?

This package is used ALL OVER. We'd have to bring the dep back into the Dart SDK, etc